testing/tests/DevExpress.ui.widgets/overlay.tests.js

```diff
@@ -2154,44 +2159,21 @@ testModule('close on target scroll', moduleConfig, () => {
         });
         const overlay = $overlay.dxOverlay('instance');
 
-        const targetParent = $target.parent().get(0);
+        const containerParent = $container.parent().get(0);
         $target.detach();
         overlay.hide();
-        const parentEvents = $._data(targetParent).events || {};
+        const parentEvents = $._data(containerParent).events || {};
         assert.strictEqual('scroll' in parentEvents, false, 'scroll unsubscribed');
     });
 
-    test('all opened overlays should be closed on scroll', function(assert) {
```

Comment:
Why do we delete this test?

Reply:
see comment about https://github.com/DevExpress/DevExtreme/pull/18385/files#r682836440

Reply:
I returned it back